lib/taiko.js

```diff
@@ -2552,6 +2552,10 @@ module.exports.text = text;
 
 function text(text, options = {}, ...args) {
   validate();
+  if (options instanceof RelativeSearchElement) {
```

Comment:
why not use [`prepareParameters`](https://github.com/getgauge/taiko/blob/ff81d8ebc98fcb382fd0a8d7a692ddb7d069f6ee/lib/elementWrapper/elementWrapperFactory.js#L17) which is used by other apis?

Reply:
`prepareParameters` had few extra checks. I have made the changes to keep it consistence across API's now.

Thanks for reviewing the PR